kotlin/build.gradle

```diff
@@ -123,23 +123,6 @@ rootProject.ext.defaultDokkaConfig = {
     outputDirectory = "$buildDir/docs"
 }
 
-rootProject.ext.defaultAndroidConfig = {
```

Comment:
It doesn't look like you inlined this block, or moved it to somewhere – were we just not using it?

Reply:
I moved it toward the top, s.t. the android version #s stayed prominent.